### PR TITLE
Bug 568152 - Use FrameworkUtil#getBundle instead of Platform#getBundle

### PR DESCRIPTION
**Pull request by kthoms**
Regenerated languages.

Signed-off-by: Karsten Thoms <karsten.thoms@karakun.com>

### CODE REVIEW

File: org.eclipse.xtext.ui.tests/src-gen/org/eclipse/xtext/ui/tests/editor/contentassist/services/Bug332217TestLanguageGrammarAccess.java [hunk 1]
```diff
@@ -983,10 +983,7 @@
 		private final Keyword cSubtitleSubtitleKeyword_3_0 = (Keyword)cSubtitleEnumLiteralDeclaration_3.eContents().get(0);
 		
 		//enum CellType:
-		//	default='Default' |
-		//	value1='Value1' |
-		//	value2='Value2' |
-		//	subtitle='Subtitle';
+		//	default='Default' | value1='Value1' | value2='Value2' | subtitle='Subtitle';
```
Comment (kthoms) [inline org.eclipse.xtext.ui.tests/src-gen/org/eclipse/xtext/ui/tests/editor/contentassist/services/Bug332217TestLanguageGrammarAccess.java:986]: this one is unrelated to the change, but caused by regeneration. leave it?

File: org.eclipse.xtext.ui.tests/src-gen/org/eclipse/xtext/ui/tests/editor/contentassist/services/Bug332217TestLanguageGrammarAccess.java [hunk 2]
```diff
@@ -1362,10 +1359,7 @@ public ParserRule getCellRule() {
 	}
 	
 	//enum CellType:
-	//	default='Default' |
-	//	value1='Value1' |
-	//	value2='Value2' |
-	//	subtitle='Subtitle';
+	//	default='Default' | value1='Value1' | value2='Value2' | subtitle='Subtitle';
```
Comment (kthoms) [inline org.eclipse.xtext.ui.tests/src-gen/org/eclipse/xtext/ui/tests/editor/contentassist/services/Bug332217TestLanguageGrammarAccess.java:1362]: this one is unrelated to the change, but caused by regeneration. leave it?

File: org.eclipse.xtext.xbase.ui.tests/META-INF/MANIFEST.MF [hunk 3]
```diff
@@ -32,3 +32,5 @@ Import-Package: org.apache.log4j;version="1.2.15",
  org.junit.runner;version="4.5.0",
  org.junit.runners;version="4.5.0"
 Automatic-Module-Name: org.eclipse.xtext.xbase.ui.tests
+Export-Package: org.eclipse.xtext.xbase.testlanguages.ui.tests;x-internal=true,
```
Comment (kthoms) [inline org.eclipse.xtext.xbase.ui.tests/META-INF/MANIFEST.MF:35]: this one is unrelated to the change, but caused by regeneration. leave it?

Comment (cdietrich) [reply org.eclipse.xtext.xbase.ui.tests/META-INF/MANIFEST.MF:35]: who generates that?

Comment (cdietrich) [reply org.eclipse.xtext.xbase.ui.tests/META-INF/MANIFEST.MF:35]: i dont have this change

File: org.eclipse.xtext.ui.tests/src-gen/org/eclipse/xtext/ui/tests/editor/contentassist/services/Bug332217TestLanguageGrammarAccess.java (hunk 1 above)
Comment (cdietrich) [reply org.eclipse.xtext.ui.tests/src-gen/org/eclipse/xtext/ui/tests/editor/contentassist/services/Bug332217TestLanguageGrammarAccess.java:986]: yes keep

File: org.eclipse.xtext.ui.tests/src-gen/org/eclipse/xtext/ui/tests/editor/contentassist/services/Bug332217TestLanguageGrammarAccess.java (hunk 2 above)
Comment (cdietrich) [reply org.eclipse.xtext.ui.tests/src-gen/org/eclipse/xtext/ui/tests/editor/contentassist/services/Bug332217TestLanguageGrammarAccess.java:1362]: yes keep

File: org.eclipse.xtext.xbase.ui.tests/META-INF/MANIFEST.MF (hunk 3 above)
Comment (cdietrich) [reply org.eclipse.xtext.xbase.ui.tests/META-INF/MANIFEST.MF:35]: (also which workflow)

Comment (cdietrich) [reply org.eclipse.xtext.xbase.ui.tests/META-INF/MANIFEST.MF:35]: maybe this one was not regenerated for a longer time.

**Review by cdietrich — APPROVED** [review]

**Review by szarnekow — APPROVED** [review]

File: org.eclipse.xtext.xbase.ui.tests/META-INF/MANIFEST.MF (hunk 3 above)
Comment (kthoms) [reply org.eclipse.xtext.xbase.ui.tests/META-INF/MANIFEST.MF:35]: This is added when executing xtext-extras/org.eclipse.xtext.xbase.testlanguages/src/org/eclipse/xtext/xbase/testlanguages/GenerateXbaseTestLanguages.mwe2

Reproduced on master

Comment (cdietrich) [reply org.eclipse.xtext.xbase.ui.tests/META-INF/MANIFEST.MF:35]: indeed 👍

**Pull request merged**